v2/yarpcjson/register.go

```diff
@@ -53,6 +53,21 @@ func Procedure(name string, handler interface{}) []yarpc.TransportProcedure {
 	}
 }
 
+func procedure(name string, handler interface{}) []yarpc.EncodingProcedure {
+	return []yarpc.EncodingProcedure{
+		{
+			Name: name,
+			HandlerSpec: yarpc.NewUnaryEncodingHandlerSpec(
+				jsonHandler2{
```

Comment:
Perhaps jsonEncodingHandler as opposed to the existing jsonTransportHandler, though that one goes away and this one simplifies back regardless. We’re not picky about temporary names, so whatever you like.

Reply:
Of course, this is definitely not the intended final name!!

I was actually thinking it wouldn't be too much of a change to replace the existing yarpcjson.Procedure() with this in the same PR. What do you think?